inductiva/resources/machines_base.py

```diff
@@ -92,6 +88,13 @@ def __init__(
     def id(self):
         return self._id
 
+    @property
+    def vcpus(self):
+        # This works for GCP and ICE
+        # TODO: Add support for other providers
+        return self.quota_usage.get("total_num_vcpus",
+                                    int(self.machine_type.split("-")[2]))
```

Comment:
Running this code 
```
cluster = inductiva.resources.MPICluster(machine_type="c2-standard-4", num_machines=2)
mg = inductiva.resources.MachineGroup(machine_type="c2-standard-4", num_machines=2)
cluster.start()
mg.start()
```
Cluster has this quotas `{'max_vcpus': 8.0, 'max_instances': 2.0, 'max_price_hour': 0.46496}`
MachineGroup has this quotas `{'max_vcpus': 8.0, 'max_instances': 2.0, 'max_price_hour': 0.11672}`

What should vcpus return when running on a machine group? max_vcpus // max_instances?
On the cluster it would make sense to me to just return the max_vcpus no ?

Reply:
For machine base we would have
```python
@property
    def vcpus(self):
        # This works for GCP and ICE
        # TODO: Add support for other providers
        return (self.quota_usage["max_vcpus"] //
                self.quota_usage["max_instances"])
```
Then we just need to override the method on machine_clusters with 
```python
    @property
    def vcpus(self):
        # This works for GCP and ICE
        # TODO: Add support for other providers
        return self.quota_usage["max_vcpus"]
```